examples/degu-image.js

```diff
@@ -1,9 +1,9 @@
 
- import { DeguImage} from '../lib/lit/image';
+ import { register} from '../lib/lit/lit';
 
 export default class DeguImageSample {
 
   constructor() {
-    window.customElements.define('degu-image', DeguImage);
+    register();
```

Comment:
I'm not seeing the value to this at this time (but may after the library grows?). I think users may still have some value in enumerating exactly which components they want to use in their projects (versus bringing all of them in).

One nit here is I would name this `defineAll` (or similar) so it's clearer what it's doing.

Thoughts @stevenle ?

Reply:
I do like the idea of being able to pick and choose which components since a project may only use a couple, especially if we eventually have very complex components that are more project-specific (like a degu-lottie-player components which not a lot of sites will use).

I'm fine with `defineAll` but I would suggest having something like "Components" (e.g. `defineAllComponents` or `defineComponents`) in the name of the method since it helps readability when it's imported and used in another file.

Reply:
@jeremydw Related to the comments above, I think the plugin system you're creating with ama should have some way of defining its degu component dependencies so that a caller of the plugin will just need to do something like the following instead of having to read some README to figure out which degu components are required dependnecies.

```typescript
import {FooPlugin} from './ama/plugins/foo';
FooPlugin.defineAllComponents();
```

Reply:
I can figure this out with the plugin; but note that even in the plugin we wouldn't want to define all components since only some may be used. I'll defer to working on this in the plugin until if/when we move it to a package